services/auth.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var passport                = require('passport');
+var FacebookStrategy        = require('passport-facebook');
+
+var userController          = require('../controllers/user');
+var log                     = require('../log');
+
+const facebookCfg = {
+    clientID: process.env.DAKDAK_FB_APP_ID,
+    clientSecret: process.env.DAKDAK_FB_APP_SECRET,
+    callbackURL: 'http://localhost:5000/auth/facebook/callback' // TODO parameterize
+};
+
+// these are required for Passport to work
+passport.serializeUser((user, done) => done(null, user));
+passport.deserializeUser((user, done) => done(null, user));
+
+passport.use(new FacebookStrategy(facebookCfg, (accessToken, refreshToken, profile, done) => {
```

Comment:
Line is too long.<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).